core/templates/css/oppia.css

```diff
@@ -4649,6 +4648,22 @@ div.oppia-issues-learner-action {
 
 .mat-card-content.oppia-profile-portfolio-container {
   background-color: transparent;
+  /* Increase no. of columns on contributors profile page */
```

Comment:
Unneeded comment IMO

Reply:
removed this and modified comment to be more readable.